resource-service/pom.xml

```diff
@@ -32,18 +32,20 @@
         <hadoop.version>3.2.0</hadoop.version>
         <fabric8.version>4.3.0</fabric8.version>
         <protobuf.version>3.5.1</protobuf.version>
+        <jetcd.core>0.3.0</jetcd.core>
     </properties>
 
     <dependencies>
-        <dependency>
```

Comment:
Removing this is likely to break ServiceInstanceRestController. Even though it doesn't call any Feign clients.